.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,51 @@
+name: Publish Python 🐍 distribution 📦 to PyPI
+on:
+  release:
+    types: [published]
```

Comment:
NIT: It would be nice to have a way to test the workflow. Perhaps we could have a manual trigger that publishes to a test PyPi environment

Reply:
done